src/main/resources/config/application-artemis.yml

```
@@ -80,10 +80,11 @@ artemis:
         artemis-authentication-token-key: <credentials key>
         # The actual value of the notification token to check against in Artemis. This is the token that gets send with
         # every request the CI system makes to Artemis containing a new result after a build.
-        # Bamboo: The token value you use for the Server Notification Plugin
+        # Bamboo, GitLab CI: The token value you use for the Server Notification Plugin
         # Jenkins: The token value you use for the Server Notification Plugin and is stored under the notification-token credential above
         artemis-authentication-token-value: <token>
         build-timeout: 30 # Does cancel jenkins builds after 30 minutes to remove build that get stuck
+        notification-plugin: "ls1tum/artemis-notification-plugin:pr-1" # Docker image for the generic notification plugin. This value is set in an CI variable in GitLab CI.
```

```suggestion
        notification-plugin: "ls1tum/artemis-notification-plugin:latest" # Docker image for the generic notification plugin. This value is set in an CI variable in GitLab CI.
```
As the PR is merged, the image could be changed here.

---

We are going to create a release for the notification plugin, so I will use the image of the release.